Command/AtoumCommand.php

```diff
@@ -121,7 +121,27 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $reportCli->addWriter($writerCli);
         }
 
-        $runner->run($this->getAtoumArguments());
+        try {
+            $score = $runner->run()->getRunner()->getScore();
```

Comment:
Let me fix it! Thanks for you feedback.

I'll update the PR, merge it and issue a new release ;)